content/stub.xhtml

```diff
@@ -18,11 +18,6 @@
     href="chrome://conversations/skin/tokeninput.css" />
   <link rel="stylesheet" type="text/css" media="print"
     href="chrome://conversations/skin/print.css" />
-  <script type="text/javascript">
-    <![CDATA[
-      document.allowUnsafeHTML = true;
```

Comment:
xref https://github.com/protz/thunderbird-conversations/issues/1255#issuecomment-379575588

Also https://bugzilla.mozilla.org/show_bug.cgi?id=1450315#c8

Basically, we need to move away from jQuery now. I've just filed #1317 to track that.

Reply:
Dropped jQuery features related to `innerHTML`, but this still dependents on jQuery.